openapi.yml

```diff
@@ -5890,32 +6233,6 @@ components:
     BlankEnum:
       enum:
       - ''
-    CVEv5PackageVersions:
-      type: object
-      description: CVEv5 package versions serializer
-      properties:
-        package:
-          type: string
-          maxLength: 2058
-        versions:
-          type: array
-          items:
-            $ref: '#/components/schemas/CVEv5Versions'
-      required:
-      - package
-      - versions
-    CVEv5Versions:
-      type: object
-      description: CVEv5 Package Version Serializer
-      properties:
-        version:
-          type: string
-          maxLength: 1024
-        status:
-          $ref: '#/components/schemas/StatusEnum'
-      required:
-      - status
-      - version
```

Comment:
This will most likely break compatibility, but I am not 100% sure.

Reply:
`oasdiff` check is fine. We'll see for sure once it's rebased and partially squashed tomorrow.

Reply:
It's fine.